{
  "id": "uno-ooo-parser",
  "version": "0.4.0",
  "libre_office_ver": "2024.2",
  "name": "ExecutableDialogException",
  "type": "exception",
  "namespace": "com.sun.star.ui.dialogs",
  "parser_args": {
    "sort": false,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "allow_db": true,
    "from_imports": [
      [
        "...uno.exception",
        "Exception",
        "Exception_85530a09"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.uno.Exception": "Exception_85530a09"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "full_imports": {
      "general": [
        "com.sun.star.uno.Exception"
      ],
      "typing": []
    },
    "imports": [],
    "name": "ExecutableDialogException",
    "namespace": "com.sun.star.ui.dialogs",
    "extends": [
      "com.sun.star.uno.Exception"
    ],
    "desc": [
      "Base class of all XExecutableDialog related exceptions."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1ui_1_1dialogs_1_1ExecutableDialogException.html",
    "items": {}
  }
}